From 57872430bba5228024b5089f3daa78935414a567 Mon Sep 17 00:00:00 2001
From: Jamie Gibbons <jamie.gibbons@microchip.com>
Date: Fri, 13 Jun 2025 11:23:03 +0100
Subject: [PATCH 1/2] drivers: mailbox: mpfs-mbox: add missing include

Implicit declaration of function dev_set_priv() compilation warning due
to a missing include. Add missing include to fix warning.

Signed-off-by: Jamie Gibbons <jamie.gibbons@microchip.com>
---
 drivers/mailbox/mpfs-mbox.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mailbox/mpfs-mbox.c b/drivers/mailbox/mpfs-mbox.c
index 699fdfd2..55238847 100644
--- a/drivers/mailbox/mpfs-mbox.c
+++ b/drivers/mailbox/mpfs-mbox.c
@@ -10,6 +10,7 @@
 
 #include <asm/io.h>
 #include <dm.h>
+#include <dm/device-internal.h>
 #include <dm/device.h>
 #include <dm/device_compat.h>
 #include <dm/devres.h>
-- 
2.39.5

